src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -228,12 +228,7 @@ public string NicknameFromFunctionSignatureHint(string functionSignature)
                 newName = priorNameHints[qualifiedFunction].UpgradeName;
             }
 
-            splitted = newName.Split('.');
-
-            if (splitted.Length < 2)
-                return null;
-
-            return splitted[splitted.Length - 2] + "." + splitted[splitted.Length - 1];
+            return newName;
```

Comment:
I think this will change the "nickname" that is saved into .dyns for all other nodes, will that have bad consequences? (maybe I am incorrect)